src/block/feature/style.js

```diff
@@ -91,7 +91,7 @@ export const createStyles = props => {
 	// Image.
 	if ( ! show.featuredImageAsBackground ) {
 		styles.push( {
-			...( ! show.featuredImageAsBackground ? createImageStyleSet( 'image%s', 'ugb-img', props.attributes, { inherit: false } ) : {} ),
+			...( ! show.featuredImageAsBackground ? createImageStyleSet( 'image%s', 'ugb-img', props.attributes ) : {} ),
```

Comment:
Changing the `inherit` would affect other styles including the width and height of the image.

The better approach would be to add in the correct inheritance of the `border-radius` style on mobile.